Library/Homebrew/tap.rb

```diff
@@ -14,7 +15,9 @@
 # {#user} represents the GitHub username and {#repo} represents the repository
 # name without the leading `homebrew-`.
 class Tap
-  extend Cachable
+  extend Cached
+  include Cached::Clear
+  extend Cached::Clear
```

Comment:
Would it be possible to use the included or extended callback to simplify this a little bit?

Ideally all that'd be necessary here would be to add `include Cached` or `extend Cached` and everything would be set up.

Reply:
It may be possible, but does Sorbet support following this?

Reply:
No idea. @dduugg do you happen to know if this works as expected? We do use the included callback in one part of the code but the methods are generated so they are not typed by Sorbet.

https://github.com/Homebrew/brew/blob/6a9c9c00a028cf0bcf7da5a9e5b28c861a981ebd/Library/Homebrew/extend/on_system.rb#L142-L162

Reply:
I won't have the bandwidth to look at this specific code in detail anythime, but often the answer to making `included` hook functionality work in sorbet is to use `mixes_in_class_methods`: https://sorbet.org/docs/abstract#interfaces-and-the-included-hook